pkg/models/repository_configuration.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+type RepositoryConfiguration struct {
+	Base
+	Name      string `json:"name" gorm:"default:null"`
+	URL       string `json:"url" gorm:"default:null"`
+	Version   string `json:"version" gorm:"default:null"`
+	Arch      string `json:"arch" gorm:"default:null"`
+	AccountID string `json:"account_id" gorm:"default:null"`
```

Comment:
Can you add an OrgId here too, as a string?

Reply:
can Orgid be null?

Reply:
nope, both should be non-null